client/src/main/scala/com/all4journey/webapp/pages/PersonalInfoFormJsImpl.scala

```diff
@@ -132,17 +128,13 @@ object PersonalInfoFormJsImpl extends PersonalInfoFormJs  with AddressTypePickle
                       false
                   }
 
-                  val address = AddressForm.buildObjectFromForm()
-
-                  val addressViolations = Address.doValidation(address)
-                  AddressForm.setViolationPrompts(addressViolations)
 
-                  if (validUser && addressViolations.isEmpty) {
-                    val personalFormPayload = new PersonalFormData(user, Some(address), Seq[State]())
+                  if (validUser) {
+                    val personalFormPayload = new PersonalFormData(user)
                     val pickledPfp = Pickle.intoString(personalFormPayload)
 
 
-                    AjaxHelper.doAjaxPostWithJson("/multiformProfile/personal", pickledPfp, "", AddressForm.refreshUuid, HtmlHelper.showErrorBanner)
+                    AjaxHelper.doAjaxPostWithJson("/multiformProfile/personal", pickledPfp, "", (Any) => HtmlHelper.showSuccessBanner(), HtmlHelper.showErrorBanner)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Field name '(Any)' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/rjkernick/all4journey/file/2737533537/issues/source?bid=3246460&fileBranchId=3352852#l137)